playwright/element-web-test.ts

```diff
@@ -102,7 +103,7 @@ export const test = base.extend<
     },
 
     startHomeserverOpts: "default",
-    homeserver: async ({ request, startHomeserverOpts: opts }, use) => {
+    homeserver: async ({ request, startHomeserverOpts: opts }, use, testInfo) => {
```

Comment:
shouldn't testInfo have a type annotation? (I'm surprised typescript doesn't complain)

Reply:
No as the `extend` method is well-typed:

<img width="881" alt="image" src="https://github.com/matrix-org/matrix-react-sdk/assets/2403652/0bbd80e5-3e57-4a4a-b509-fc137f59959f">